src/Service.Tests/Authorization/REST/RestAuthorizationHandlerUnitTests.cs

```diff
@@ -368,12 +368,7 @@ private static AuthorizationResolver SetupAuthResolverWithWildcardOperation()
             RuntimeConfig runtimeConfig = AuthorizationHelpers.InitRuntimeConfig(
                 entityName: AuthorizationHelpers.TEST_ENTITY,
                 roleName: "admin",
-                operation: Config.Operation.All);
-
-            // Override the operation to be a list of string for wildcard instead of a list of object created by InitRuntimeConfig()
-            //
-            runtimeConfig.Entities[AuthorizationHelpers.TEST_ENTITY].Permissions[0].Operations = new object[] { JsonSerializer.SerializeToElement(AuthorizationResolver.WILDCARD) };
-
+                operation: EntityActionOperation.All);
```

Comment:
With the removed override where list of strings was created, the function summary doesnt apply anymore, do we really need this function given that it is not setting it up with wildcard anymore? 

How do we handle representation of `*` in the config file action or include - while serializing or deserializing it? Could you point to an example on how we tackle that?

Reply:
The serialization/deserialization is handled by `EnumMemberJsonEnumConverterFactory` in which it looks at the members of the enum _as well as_ the `EnumMember` attribute to generate a map of all possible ways an enum could be named in our system, and then we write based on either its name or its attribute.